management/src/test/java/org/ehcache/management/providers/statistics/StatsUtil.java

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
I will put it back into test code, but I will probably have to duplicate some code in two projects.